lib/typescript/httpclient/index.ts

```diff
@@ -143,6 +144,13 @@ export class HttpClient {
     }
   }
 
+  public async getConversationInfo(conversationId: string) {
+    const conversation: ConversationPayload = await this.doFetchFromBackend('conversations.info', {
+      conversation_id: conversationId,
```

Comment:
camelCase?

Reply:
Not here, this is the actual call to the backend. Or did the backend change to camelCase?

Reply:
ups sorry, you are right